### PR TITLE
Add store events to typings

### PR DESCRIPTION
**Pull request by stmegabit**
Fix typings for #1450

### CODE REVIEW

**Comment by stmegabit** [comment]
Check, please

File: index.d.ts [hunk 1]
```diff
@@ -680,8 +680,19 @@ export interface Interpolator {
...
 export class ResourceStore {
   constructor(data: Resource, options: InitOptions);
+
   public data: Resource;
   public options: InitOptions;
+
+  /**
+   * Gets fired when resources got added
+   */
+  on(event: 'added', callback: (lng: string, ns: string) => void): void;
```
Comment (rosskevin) [inline index.d.ts:690]: Sorry, I just noticed, why not combine these as:

```diff
- on(event: 'added', callback: (lng: string, ns: string) => void): void;
- on(event: 'removed', callback: (lng: string, ns: string) => void): void;
+ on(event: 'removed' | 'added', callback: (lng: string, ns: string) => void): void;
```

Comment (stmegabit) [reply index.d.ts]: Thanks, fixed

**Comment by rosskevin** [comment]
@jamuhl this can be a patch release

**Comment by jamuhl** [comment]
was published in i18next@19.4.5

thanks to all 👍

**Pull request merged**